NSubstitute.Analyzers.All.sln

```diff
@@ -0,0 +1,99 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
This solution contains all projects - including Vsix, which are not supported on Linux machines